File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/exception/InvalidDateTime.java
package org.opencds.cqf.cql.engine.exception;

public class InvalidDateTime extends CqlException {
    private static final long serialVersionUID = 1L;

    public InvalidDateTime(String message) {
        super(message);
    }

    public InvalidDateTime(String message, Throwable cause) {
        super(message, cause);
    }
}

File: Src/java/engine/src/commonMain/kotlin/org/opencds/cqf/cql/engine/exception/InvalidDateTime.kt
package org.opencds.cqf.cql.engine.exception

class InvalidDateTime : CqlException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID = 1L
    }
}
